iOS/Flows/Conversation/Threads/ThreadViewController.swift

```diff
@@ -113,7 +120,10 @@ class ThreadViewController: DiffableCollectionViewController<ConversationSection
 
     override func getAllSections() -> [ConversationSection] {
         if let channelId = self.parentMessage.cid {
-            return [ConversationSection(sectionID: channelId.description,
+            let placeholderSection = ConversationSection(sectionID: channelId.description,
+                                                        parentMessageID: "Placeholder")
+            return [placeholderSection,
+                    ConversationSection(sectionID: channelId.description,
```

Comment:
A little hacky:

The time machine expects there to be a top and bottom section, but the replies only have one section. Throw a placeholder section in there so the replies show up in the bottom section.